flow/skia_gpu_object.cc

```diff
@@ -39,6 +43,11 @@ void SkiaUnrefQueue::Drain() {
   for (SkRefCnt* skia_object : skia_objects) {
     skia_object->unref();
   }
+
+  if (context_ != nullptr) {
```

Comment:
Nit: The unref queue drains all objects after `drain_delay_` but the task is posted on every GPU object unref. So it is possible to for this method to not actually unref any objects because they were cleared on a previous unref. So its good to only perform deferred cleanup if `skia_objects.size() > 0`.

Reply:
Do we track SkImage objects in this queue?  I thought this was only for GPU context objects?

Reply:
Yes. SkImages are GPU context related. We also track other objects that may themselves reference SkImages as well (like pictures).

Reply:
Done.